samples/connect-wpf/NuGet.config

```diff
@@ -3,8 +3,13 @@
   <solution>
     <add key="disableSourceControlIntegration" value="true" />
   </solution>
+  
   <packageSources>
     <clear />
-    <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
Is this change intentional or just something you were testing? cc @AbhitejJohn in case there is any policy around feeds that should be used in the dotnet repo.

Reply:
I was testing out a package version that's not in the dotnet-public repo but this can be reverted.

Reply:
It's a sample and the key reason it needs to be here is because I wanted to override and decouple from the Directory.Packages.props in the root folder.